### PR TITLE
feat(kucoinfutures) - .features

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: ts/src/test/static/request/kucoinfutures.json [hunk 1]
```diff
@@ -182,7 +182,7 @@
             {
                 "description": "Swap private trades",
                 "method": "fetchMyTrades",
-                "url": "https://api-futures.kucoin.com/api/v1/fills?symbol=LTCUSDTM&startAt=1699457638000",
+                "url": "https://api-futures.kucoin.com/api/v1/fills?symbol=LTCUSDTM&startAt=1699457638000&pageSize=5",
```
Comment (carlosmiei) [inline ts/src/test/static/request/kucoinfutures.json:185]: hum what is this?

**Comment by carlosmiei** [comment]
also build is broken here:
image

File: ts/src/test/static/request/kucoinfutures.json (hunk 1 above)
Comment (ttodua) [reply ts/src/test/static/request/kucoinfutures.json:185]: it was not implemented in fetchTrades, and i've added support for `limit` param

Comment (carlosmiei) [reply ts/src/test/static/request/kucoinfutures.json:185]: @ttodua but why 5? that seems too small

**Comment by ttodua** [comment]
> also build is broken here: image

fixed

File: ts/src/test/static/request/kucoinfutures.json (hunk 1 above)
Comment (ttodua) [reply ts/src/test/static/request/kucoinfutures.json:185]: the test was using `5` as limit , it's just random static test

Comment (ttodua) [reply ts/src/test/static/request/kucoinfutures.json:185]: i mean, if you look at 5 lines below, you'll see 'input' params

Comment (carlosmiei) [reply ts/src/test/static/request/kucoinfutures.json:185]: image
 oh right, sorry my bad, I was assuming this was the "default" limit we were sending

**Pull request merged**